src/components/Notifications/NotificationFeed/NotificationFeed.js

```diff
@@ -289,13 +294,10 @@ class NotificationFeed extends React.Component {
       })
   }
   filterNotificationsByProject = () => {
-    const limit =
-      (this.state.notificationList?.length || this.NOTIFICATION_FETCH_LIMIT) + (this.state.unFetchedNotifications ?? 0)
-
     fetchNotificationFeed({
       ...getAuthentication(this.props.authentication),
       offset: 0,
-      limit,
+      limit: 10,
```

Comment:
Why is the limit defaulting to 10 here?

Reply:
I will change it to the this.NOTIFICATION_FETCH_LIMIT